cheddar/cheddar-rest/src/main/java/com/clicktravel/cheddar/rest/media/MediaTypes.java

```diff
@@ -16,8 +16,12 @@
  */
 package com.clicktravel.cheddar.rest.media;
 
+import javax.ws.rs.core.MediaType;
+
 public class MediaTypes {
 
     public static final String CDM_V1_JSON = "application/vnd.clicktravel.schema-v1+json";
```

Comment:
We need this media type whenever we need CDM, so why not put this in CDM instead of in Cheddar. I've found in many places that this value is redefined e.g. service tests, end-to-end test runner etc. as those projects do not have access to this as a dependency.

In summary, I don't feel these values belong in Cheddar.